docs/installation.md

```diff
@@ -115,10 +115,16 @@ Once you have Docker installed, simply create the config file (e.g. `config.json
 
 **1.1. Clone the git repository**
 
+Linux
 ```bash
 git clone https://github.com/freqtrade/freqtrade.git
 ```
 
+Windows (docker or WSL)
```

Comment:
is this also a problem if you run `git clone` in WSL?

I'm not 100% certain but I think git inside WSL should handle linefeeds correctly since it's a ubuntu (or other linux) binary.

will need to check that ...

Reply:
i will make the test to make tomorrow think you are right no need to redefined something that is correctly defines.

Reply:
i have test no need to to this on WSL